src/integration-test/java/uk/gov/ida/verifylocalmatchingserviceexample/rules/TestDatabaseRule.java

```diff
@@ -32,12 +36,16 @@ protected void after() {
     }
 
     private void setUpDatabase() {
-        handle.execute("create table verifiedPid(pid TEXT)");
+        Flyway flyway = new Flyway();
+        flyway.setDataSource(this.appRule.getConfiguration().getDataSourceFactory().getUrl(), this.appRule.getConfiguration().getDataSourceFactory().getUser(), this.appRule.getConfiguration().getDataSourceFactory().getPassword());
+        flyway.setLocations("classpath:db.migration.common", this.appRule.getConfiguration().getDatabaseMigrationSetup().getDatabaseEngine().getEngineSpecificMigrationsLocation());
+        flyway.clean();
+        flyway.migrate();
     }
 
     public void ensurePidExist(String verifiedPid) {
         handle.begin();
-        handle.execute("insert into verifiedPid values('" + verifiedPid + "')");
+        handle.execute("insert into verifiedPid (pid, person) values('" + verifiedPid + "', (select person_id from person limit 1))");
```

Comment:
This is essentially against a random person (as order by is not specified), which seems fine. This syntax works in both postgresql and sqlite, and probably other engines (but not tested in those).